gk/main.c

```diff
@@ -552,32 +553,17 @@ gk_del_flow_entry_from_hash(struct rte_hash *h, struct flow_entry *fe)
 	return ret;
 }
 
-/* Return true when it removes at least one entry, and false otherwise. */
+/* Return true when it removes an entry, and false otherwise. */
 static bool
-gk_flow_tbl_bucket_scan(uint32_t *bidx, struct gk_instance *instance)
+gk_flow_tbl_entry_scan(uint32_t entry_idx, struct gk_instance *instance)
```

Comment:
I wonder have we considered to scan a batch of entries instead of one entry each time?